### PR TITLE
Fixes some of the test

### PR DESCRIPTION
**Pull request by keshav234156**
Fixes Some of the #1285

### CODE REVIEW

**Comment by keshav234156** [comment]
Oh! I just saw the coverage report. Coverage would almost be doubled and at all time maxima. @jywarren We are on the right path but need to work a harder on commented test module that was preventing other modules

File: test/core/modules/text-overlay.js [hunk 1]
```diff
@@ -24,19 +24,19 @@ test('Check Options', function(t) {
...
-    red = './test_outputs/red.png';
-    looksSame(result, red, function(err, res) {
-      if (err) console.log(err);
-      t.equal(res.equal, true);
-      t.end();
-    });
-  });
-});
\ No newline at end of file
+// test('Text Overlay module works correctly', function(t) {
```
Comment (pythongiant) [inline test/core/modules/text-overlay.js]: Can we use multi line comments here? `/* */` Just a suggestion

**Comment by keshav234156** [comment]
@jywarren I think we should merge this so that our other test are not affected.what your view on this!!!

**Comment by jywarren** [comment]
Yes, ok let's merge this as soon as it passes!

**Comment by jywarren** [comment]
Can you make the PR title a little more descriptive? It's good to leave a more thorough history! Thanks a lot!!!

**Comment by keshav234156** [comment]
@jywarren our next step should be fix 7 modules that i have mentioned in #1285 .For that i found the module were changed as it has some bugs like the rotate module was earlier having fixed dimension but corrosponding benchmarks were not changed as a result they are failing.So for this we can change the benchmarks of that modules.Then our next step should be if there is any failure in test then travis should raise an error rather than passing it .What your view 's regarding this?

**Comment by keshav234156** [comment]
@jywarren Would take care of descriptive title from now on.Thanks!!!

**Comment by jywarren** [comment]
Thank you, that's a fantastic plan. And please copy your comment back into #1285 so we can continue tracking progress in one place -- thanks a ton!!!

**Pull request merged**